### ISSUE TITLE
Change location of a synced folder

### ISSUE THREAD

**Issue by xerus2000** [Catfriend1/syncthing-android#763]
I accidentally added a folder directly in the data-dir of the application (there should probably be a warning, I intuitively expected it to create a subfolder).

To move it, it seems I currently have to delete and re-add and thus re-configure everything, which is very unhandy :/

**Comment by Catfriend1** [comment]
No, reason explained by the core devs: https://forum.syncthing.net/t/why-cant-we-change-the-folder-paths/16507/28
Syncthing-Fork will follow this advice NOT to add the feature to edit the folder path via UI.

**Comment by xerus2000** [comment]
hm, I would expect it to
1) pause syncing
2) move the contents of the folder to the new location
3) change the folder location
4) continue syncing
Is there some more elaboration at a specific point in the thread? I don't fancy looking through it all.

**Comment by Catfriend1** [comment]
No, syncthing is not a file mover!

**Comment by Catfriend1** [comment]
And I don't think you'd want to write SAF/native/all files access code for doing moves across all android versions as it's handled different by google's restrictions.

**Comment by xerus2000** [comment]
is there any way to change the location of a folder without adding it completely new again?

**Comment by Catfriend1** [comment]
I've did this on my phone a while ago. If you make a mistake, your data will be toasted. Better do a backup first.

If you do it exactly like this:
- Ensure everything is in-sync
- Export Syncthing config
- Stop Syncthing App (Exit, then force-stop)
- Edit the folder path in /storage/emulated/0/backups/config.xml , Save.
- Move the folder to the new location (do NOT change metadata like lastModTime, file attributes, permissions etc.)
- Double check, you did everything right and made no typos.
- Start Syncthing App (It will report a stopped folder that does not exist.)
- Import Syncthing config
